### DISCUSSION TITLE
How to disable triger of diktat on tests with gradle?

### DISCUSSION THREAD

**Discussion by orchestr7** [cqfn/diKTat#956] [Q&A] [answered]
Could you please suggest: how to disable diktat triggerring on tests?

**Comment by petertrr** [comment, marked as answer]
There is a property `exclude` in `DiktatExtension`. You can find it's usage examples, e.g., in our tests:
https://github.com/cqfn/diKTat/blob/93bfd084c6c83d0a674f0d3a2128c43ca2e31937/diktat-gradle-plugin/src/test/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskTest.kt#L42-L52
